ergo-lib/src/wallet/tx_builder.rs

```diff
@@ -199,8 +200,10 @@ impl<S: ErgoBoxAssets + ErgoBoxId + Clone> TxBuilder<S> {
             ));
         }
         // check that inputs have enough tokens
-        let input_tokens = sum_tokens_from_boxes(self.box_selection.boxes.as_slice());
-        let output_tokens = sum_tokens_from_boxes(output_candidates.as_slice());
+        let input_tokens = sum_tokens_from_boxes(self.box_selection.boxes.as_slice())
+            .map_err(TxBuilderError::TokenAmountError)?;
+        let output_tokens = sum_tokens_from_boxes(output_candidates.as_slice())
+            .map_err(TxBuilderError::TokenAmountError)?;
```

Comment:
The same `map_err` -> `?` is applicable here.

Reply:
Oops, sorry I missed that.